exercises/rna-transcription/src/test/java/RnaTranscriptionTest.java

```diff
@@ -4,6 +4,9 @@
 
 import static org.junit.Assert.assertEquals;
 
+/*
+ * Based on problem-specifications/exercises/rna-transcription/canonical-data.json 1.3.0
+ */
```

Comment:
This comment isn't needed. We've got the version file to tell us which version the tests and reference solution are based on 🙂

Reply:
Removed!